src/test/java/org/broadinstitute/hellbender/utils/read/AlignmentUtilsUnitTest.java

```diff
@@ -187,7 +187,7 @@ private GATKRead makeReadForAlignedToRefTest(final String baseString) {
             badHap.setAlignmentStartHapwrtRef(hapStart);
             final Haplotype refHap = makeHaplotypeForAlignedToRefTest(ref, ref.length() + "M");
 
-            final int expectedPos = 13684; // = 13011 + 553 + 120 (alignemnt offset)
+            final int expectedPos = 13684; // expectedPos = refStart + hapStart + alignmentOffset
```

Comment:
@DonFreed What I meant here was to replace the literal `13684` in the code with exactly what you have in the comment ie the *code* should be `final int expectedPos = refStart + hapStart + alignmentOffset`.